lib/glific/contacts.ex

```diff
@@ -202,9 +202,7 @@ defmodule Glific.Contacts do
   """
   @spec contact_opted_in(String.t(), DateTime.t()) :: {:ok}
   def contact_opted_in(phone, utc_time) do
-    # Still need to figure out how to do that in single query
-    upsert(%{phone: phone, optin_time: utc_time})
-
+    upsert(%{phone: phone, optin_time: utc_time, status: :valid})
```

Comment:
Does gupshup send the name here by any chance?

Reply:
No @dlobo Gupshup does not send any name on the contact option callback.

```
Parameters: %{"app" => "TidesTestApi", "payload" => %{"phone" => "919917443994", "type" => "opted-in"}, "timestamp" => 1592830187179, "type" => "user-event", "version" => 2}
```

However, we can update the contact name when we receive the message from the contact.